apps/meteor/app/e2e/client/rocketchat.e2e.ts

```diff
@@ -418,15 +418,31 @@ class E2E extends Emitter {
 			return message;
 		}
 
-		const data = await e2eRoom.decrypt(message.msg);
+		const attachmentDescription = message.attachments?.[0]?.description;
+
+		const textToDecrypt = attachmentDescription || message.msg;
```

Comment:
I don't think we should decrypt one or the other, I'd say we should keep decrypting the msg.msg AND add the ability to decrypt file descriptions as well.